src/status_im2/common/standard_auth/keycard/view.cljs

```diff
@@ -0,0 +1,178 @@
+  (:require [react-native.core :as rn]
+            [quo2.core :as quo]
+            [status-im2.common.standard-auth.keycard.style :as style]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [status-im2.common.resources :as resources]
+            [utils.i18n :as i18n]
+            [quo2.components.keycard.view :as keycard]
+            [utils.re-frame :as rf]))
+
+(defn handle-number-entry
+  [entered-numbers max-digits max-attempt-reached number]
+  (when (>= (count @entered-numbers) max-digits)
+    (reset! entered-numbers []))
+  (when (= (count @entered-numbers) (dec max-digits))
+    (reset! max-attempt-reached true))
+  (swap! entered-numbers conj number))
+
+(defn enter-keycard-pin-sheet
+  []
+  (let [entered-numbers     (reagent/atom [])
+        max-digits          6
+        max-attempt-reached (reagent/atom false)]
+
+    (fn []
+      [rn/view {:style style/container}
+       [rn/view {:style style/inner-container}
+        [quo/text
+         {:accessibility-label :enter-keycard-pin-text
+          :size                :heading-2
+          :weight              :semi-bold}
+         (i18n/label :t/enter-keycard-pin)]
+
+        [rn/view {:style style/context-tag}
+         [quo/context-tag
+          {:type    :icon
+           :icon    :i/placeholder
+           :size    24
+           :context (i18n/label :t/card-with-name {:name "Alisher"})}]]
+
+        [rn/view (style/get-digits-container-style @max-attempt-reached)
+         (for [i (range max-digits)]
+           [rn/view
+            {:key   i
+             :style (style/digit-style @max-attempt-reached i entered-numbers)}])]
+
+        (when @max-attempt-reached
+          [rn/view {:style style/max-attempt-reached-container}
+           [quo/text
+            {:size  :label
+             :style {:color colors/danger}}
+            (i18n/label :t/attempts-left {:attempts 4})]])]
+
+       [quo/numbered-keyboard
+        {:disabled?   false
+         :on-press    (fn [number]
+                        (handle-number-entry entered-numbers max-digits max-attempt-reached number))
+         :blur?       false
+         :delete-key? false
+         :left-action :none}]])))
+
+(defn this-is-not-keycard-sheet
+  []
+  [rn/view {:style style/container}
+   [rn/view {:style style/inner-container}
+    [quo/button
+     {:container-style style/close-button
+      :type            :grey
+      :icon-only?      true
+      :size            32
+      :on-press        #(rf/dispatch [:hide-bottom-sheet])} :i/close]
+
+    [quo/text
+     {:accessibility-label :this-is-not-keycard-text
+      :size                :heading-1
+      :weight              :semi-bold}
+     (i18n/label :t/this-is-not-keycard)]
+
+    [quo/text
+     {:accessibility-label :make-sure-text
+      :size                :paragraph-1
+      :weight              :medium
+      :style               style/secondary-text} (i18n/label :t/make-sure-scanned-keycard)]]
```

Comment:
https://guide.clojure.style/#vertically-align-fn-args

Please, check other places in the PR where this style guide rule needs to be applied. This style rule is very important in lisps, otherwise it becomes really hard to read the code.

Reply:
@ilmotta Did review the guide and made the needed update here and for other places in this file.